python/cli/prelude_cli/views/interactive.py

```diff
@@ -909,9 +909,16 @@ def interactive(account):
             if not yes:
                 break
 
-            wizard.iam.new_account(handle=os.getlogin())
-            keychain = PurePath(Path.home(), '.prelude', 'keychain.ini')
-            print(f'Account created! Credentials are stored in your keychain: {keychain}')
+            email = Prompt.ask('Enter your email handle')
+            creds = wizard.iam.new_account(handle=email)
+
+            cfg = ctx['keychain'].read_keychain_config()
+            cfg[ctx['keychain'].profile]['account'] = creds['account_id']
```

Comment:
Could we pass around an object instead of a dict for ctx here?

Reply:
we couuld, but I am not creative enough to see a good object to pass through. The things this context object needs at some point or another:
- the SDK account object, the CLI Keychain object (both set in cli.py), the SDK controller object (set in each controller)

It's that last one that makes a single object not work perfectly for me

Reply:
I suppose I could make a CLI Account object that inits w/ the SDK account and CLI Keychain and then add a get/set_controller() type of functionality to it...

sounds a little overkill to me, but can try it if you think it sounds better